libs/remix-tests/package.json

```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "@remix-project/remix-lib": "^0.4.31",
     "@remix-project/remix-simulator": "^0.1.9-beta.8",
-    "@remix-project/remix-solidity": "^0.3.32",
+    "@remix-project/remix-solidity": "file:../remix-solidity",
```

Comment:
this need to be rollbacked i think

Reply:
Yes, to get this PR working fine this was required. This will be rollbacked during the release process